docker-compose.yml

```diff
@@ -38,6 +38,7 @@ services:
       CHAIN_ID: "fetchchain"
     volumes:
       - ./:/app
+      - ./docker/node-cosmos/package.json:/usr/local/lib/node_modules/@subql/node-cosmos/package.json
```

Comment:
We should duplicate these changes for the ci-compose.yml and tracing-compose.yml files as well.